pkg/apis/logging/v1/elasticsearch_types.go

```diff
@@ -48,6 +48,7 @@ type ElasticsearchSpec struct {
 	RedundancyPolicy RedundancyPolicyType  `json:"redundancyPolicy"`
 	Nodes            []ElasticsearchNode   `json:"nodes"`
 	Spec             ElasticsearchNodeSpec `json:"nodeSpec"`
+	IndexManagement  *IndexManagementSpec  `json:"indexManagement"`
```

Comment:
do we want this to be able to be `nil`?

Reply:
Yes, represents undefined.  I would assume Jaeger would like to move to this impl of IndexManagement but they may not.